### PR TITLE
Header and Navbar Completed

### PR DESCRIPTION
**Pull request by AnjaniKumar1515**
» Used react to create the header and navbar 
» Styled the navbar by adding the necessary CSS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,70 @@
-# IndexGuru
+# Getting Started with Create React App
 
-"My first pull request"
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```
Comment (aditya-190) [inline README.md]: Please update the README file to add a brief description about your project and you can also add checklist of the tasks that you have done in the project for tracking your progress.

File: README.old.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# IndexGuru
+
```
Comment (aditya-190) [inline README.old.md]: You can remove this file.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "indexguru",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.5",
```
Comment (aditya-190) [inline package.json:6]: You can add the testing-libraries in devDependencies.

File: public/manifest.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```
Comment (aditya-190) [inline public/manifest.json]: Add your project name here.

File: src/Components/Login.js [hunk 5]
```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { ReactDOM } from "react";
+const App = (props) => <LoginForm />;
+
+class LoginForm extends React.Component {
```
Comment (aditya-190) [inline src/Components/Login.js:5]: In my opinion, It is better to use functional components. Read this to know more -> https://stackoverflow.com/a/57579403

File: src/Components/Navbar.js [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+  //     if (button[i].hasAttribute("id", "active")) {
+  //       button[i].removeAttribute("id", "active");
+  //     }
+  //   }
+  //   e.target.setAttribute("id", "active");
+  // };
+  const [imgNo, setImgNo] = useState(0);
+
+  useEffect(() => {
+    setInterval(() => {
```
Comment (aditya-190) [inline src/Components/Navbar.js]: Hey I am not sure why you are using `setInterval`, but if it is required, please also use `clearInterval` in useEffect return. To know more, read this -> https://www.codementor.io/@damianpereira/how-to-use-clearinterval-inside-react-s-useeffect-and-why-it-is-important-1si7mztjlk

**Review by aditya-190 — CHANGES_REQUESTED** [review]
Please make the required changes before merging the changes.

**Review by aditya-190 — APPROVED** [review]
Looks Good to me. Pls merge these changes.

File: README.old.md (hunk 2 above)
Comment (AnjaniKumar1515) [reply README.old.md]: This file was removed

File: public/manifest.json (hunk 4 above)
Comment (AnjaniKumar1515) [reply public/manifest.json]: The required changes have been made.

**Pull request merged**